temporal-sdk/src/main/java/io/temporal/internal/sync/WorkflowThreadImpl.java

```diff
@@ -249,12 +249,8 @@ public void start() {
         taskFuture = threadPool.submit(task);
         return;
       } catch (RejectedExecutionException e) {
-        getWorkflowContext()
```

Comment:
This reporting will be done inside `cache.evictAnyNotInProcessing` call down the line.